CRM/Contribute/Tokens.php

```diff
@@ -194,7 +194,43 @@ public function evaluateToken(TokenRow $row, $entity, $field, $prefetch = NULL)
       $row->tokens($entity, $field, $fieldValue);
     }
     else {
-      $row->dbToken($entity, $field, 'CRM_Contribute_BAO_Contribution', $field, $fieldValue);
+      // This section is copied & pasted & can be cleaned up later.
```

Comment:
@eileenmcnaughton is there a reason why you have done the copy n paste here?

Reply:
@seamuslee001 yeah - I always ensure the change that moves code - (which is also up separately here) https://github.com/civicrm/civicrm-core/pull/20992 only has as much in it as is required to move it. That reduces mistakes and means that you don't 'slip something in' that the reviewer would not notice

Reply:
(once all the current PRs are merged I will extend the tests to cover the rest of the tokens to complete reconciling them & rip out what I can demonstrate with a test is unnecessary)

Reply:
once https://github.com/civicrm/civicrm-core/pull/21000 is merged the copied function is entirely unused / deprecated

Reply:
Resolving this as the discussed change has been taken out of scope of this PR